contrib/graph-gophers/graphql-go/graphql_test.go

```diff
@@ -140,3 +141,51 @@ func TestAnalyticsSettings(t *testing.T) {
 		assertRate(t, mt, 0.23, WithAnalyticsRate(0.23))
 	})
 }
+
+func TestOmitTrivial(t *testing.T) {
```

Comment:
Can we somehow join this with `Test` and have 2 sub-tests to reduce duplication?

Reply:
Should be better now.